app/controllers/verify/review_controller.rb

```diff
@@ -71,7 +70,7 @@ def idv_params
 
     def phone_confirmation_required?
       idv_params[:phone] != current_user.phone &&
```

Comment:
here we are reading the value, aren't we? (just want to make sure I am understanding)